tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -8,7 +8,7 @@ d) DEVICE_TYPE=${OPTARG};;
 esac
 done
 
-PYTHON_VER=${PYTHON_VER:=3.5}
+PYTHON_VER=${PYTHON_VER:=3.6}
```

Comment:
This is the default value. I think it's safer to leave the default value alone. If you need to change it, seach 'install_ubuntu.sh' and add command line args there.

Reply:
@kit1980 please add 
ARG PYTHON_VERSION=3.6 was 3.5 in Dockerfile.ubuntu